src/Pelagos/Entity/Fileset.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Pelagos\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Fileset Entity class.
+ *
+ * @ORM\Entity
+ */
+class Fileset extends Entity
```

Comment:
Have we consider the reach back? Meaning, do you want to know which dataset submission a fileset goes with if you one have the fileset entity/id?

Reply:
I think the bidirectional relationship could be useful for the Fileset entity and File entity, but i'm not sure if we need it for this one